config/pod.yaml.tim

```diff
@@ -4,52 +4,69 @@ metadata:
   name: "{{build_id_with_prefix}}"
   labels:
     sdbuild: "{{build_id_with_prefix}}"
-    app: screwdriver
+    app: screwdriver-vm
     tier: builds
   annotations:
-    pod.alpha.kubernetes.io/init-containers: '[
-      {
-        "name": "launcher",
-        "image": "screwdrivercd/launcher:{{launcher_version}}",
-        "command": ["/bin/sh", "-c", "cp -a /opt/sd/* /opt/launcher"],
-        "volumeMounts": [
-          {
-            "name": "screwdriver",
-            "mountPath": "/opt/launcher"
-          }
-        ]
-      }
-    ]'
+   pod.alpha.kubernetes.io/init-containers: '[
+     {
+       "name": "launcher",
+       "image": "screwdrivercd/launcher:{{launcher_version}}",
+       "command": ["/bin/sh", "-c", "cp -a /opt/sd/* /opt/launcher"],
+       "volumeMounts": [
+         {
+           "name": "sdlauncher",
+           "mountPath": "/opt/launcher"
+         }
+       ]
+     }
+  ]'
 spec:
-  serviceAccount: {{service_account}}
+  tolerations:
+    - key: vm
+      value: "true"
+      effect: NoSchedule
+  nodeSelector:
+    vm: hyperd
   restartPolicy: Never
   containers:
-  - name: build
-    image: {{container}}
+  - name: docker
+    image: {{base_image}}
+    imagePullPolicy: Always
+    securityContext:
+      privileged: true
     resources:
       limits:
         memory: 2Gi
-    command:
-    - "/opt/sd/tini"
-    - "--"
-    - "/bin/sh"
-    - "-c"
-    # Run the launcher and logservice in the background
```

Comment:
The hyper-runner.sh does this part now, right? Is there anywhere I can see that? Looking at https://github.com/screwdriver-cd/screwdriver/pull/625/files I see that it *can* be configured, but not what the default is.